util/envDynawo.sh

```diff
@@ -1194,13 +1194,16 @@ build_doxygen_doc() {
 
 test_doxygen_doc() {
   if [ -f "$DYNAWO_INSTALL_DIR/doxygen/warnings.txt"  ] ; then
-    nb_warnings=$(wc -l $DYNAWO_INSTALL_DIR/doxygen/warnings.txt | awk '{print $1}')
+    rm -f $DYNAWO_INSTALL_DIR/doxygen/warnings_filtered.txt
+    # need to filter "return type of member (*) is not documented" as it is a doxygen bug detected on 1.8.17 that will be solved in 1.8.18
+    cat $DYNAWO_INSTALL_DIR/doxygen/warnings.txt | grep -v -e "return type of member .* is not documented" > $DYNAWO_INSTALL_DIR/doxygen/warnings_filtered.txt
```

Comment:
Shouldn't we make sure that we filter only the static methods? (not sure there is a way to do so though)

Reply:
The line in the doxygen warnings doesn't contain the attribute static or virtual so we cannot

Reply:
It means that all return types issue will be filtered, even the real ones... Maybe with a file that contains all the files to filter?